### DISCUSSION TITLE
Invalid Profile Public Key!

### DISCUSSION THREAD

**Discussion by malafk** [Minestom/Minestom#1627] [Help]
I didn't run MojangAuth.init() and I try to join on a cracked session and it says that, I can login with a premium account tho

**Comment by jandrokt** [comment]
I cannot replicate this issue. What version of Minestom are you running?

**Reply by malafk** [reply]
implementation 'com.github.Minestom:Minestom:-SNAPSHOT'

**Reply by jandrokt** [reply]
Are you using any proxy?

**Reply by malafk** [reply]
No

**Reply by Distractic** [reply]
Have you any log or other things to help us in order to find the problem ?
